src/content/en/progressive-web-apps/_index.yaml

```diff
@@ -27,7 +27,7 @@ landing_page:
         <ul>
           <li>
             <a href="#reliable"><b>Reliable</b></a> - Load instantly and never
-            show the downasaur, even in uncertain network conditions.
+            show the dinosaur, even in uncertain network conditions.
```

Comment:
we've intentionally used the word downasaur to indicate the offline dino you get when you're offline.